.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+version: 2.1
```

Comment:
copyright info here will be nice

Reply:
will look into adding copyright info via automation script in a future pr